lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -70,7 +70,8 @@
         'WebSiteManagementClient': '2018-02-01',
         'PostgreSQLManagementClient': '2017-12-01',
         'MySQLManagementClient': '2017-12-01',
-        'MariaDBManagementClient': '2019-03-01'
+        'MariaDBManagementClient': '2019-03-01',
+        'EventHubManagementClient': '2018-01-01-preview'
```

Comment:
is there stable version instead of the preview one?

Reply:
> is there stable version instead of the preview one?

Yeah, there is a stable version which I use. However, does the word  ```latest``` in the code fragment```AZURE_API_PROFILES = {
    'latest': {``` means the latest version of the python SDK? Or I misunderstand it that it should be the version we choose currently and I will change it.